airbyte-integrations/connectors/source-stripe/source_stripe/schemas/invoice_line_items.json

```diff
@@ -7,6 +7,9 @@
     "invoice": {
       "type": ["null", "string"]
     },
+    "invoice_id": {
```

Comment:
added new field because invoice_line_items didn't have any reference to its related invoice